### DISCUSSION TITLE
본인인증 안되시는분은 없나요?

### DISCUSSION THREAD

**Discussion by pise77** [changdoc/naver-vaccine-macro#125] [General]
해결했습니당

**Comment by changdoc** [comment]
예약 신청 창으로 안넘어가질때 개발자 도구로 console탭에 어떤 내용이 나오는지 공유가능하실까요?
저는 겪어본적이 없는 상황이라 무슨 상황인지 예측이 안되네요.

https://github.com/changdoc/naver-vaccine-macro/issues/42 동시에 예약 신청 페이지에 접근하면 계정이 한시간 동안 차단 당합니다.
